src/frontend/apps/desk/src/features/teams/components/Panel/Panel.tsx

```diff
@@ -11,32 +12,60 @@ export const Panel = () => {
   const { t } = useTranslation();
   const { colorsTokens } = useCunninghamTheme();
 
+  const [isOpen, setIsOpen] = useState(true);
+
   return (
     <Box
-      $width="100%"
+      $width={isOpen ? '100%' : '0'}
       $maxWidth="20rem"
```

Comment:
It will smooth the closing.

```suggestion
      $maxWidth={isOpen ? '20rem' : '0'}
```

Reply:
I tried to improve my check, lmk if it's okay for you